spec/src/indexSpec.js

```diff
@@ -136,6 +136,41 @@ describe('cloudwatch-metrics', function() {
       metric.put(1, 'metricName', [{Name:'ExtraDimension',Value: 'Value'}]);
       metric.put(2, 'metricName', [{Name:'ExtraDimension',Value: 'Value'}]);
     });
+
+    it('should set a Timestamp if specified in the options', function(done) {
+      attachHook(function(data, cb) {
+        expect(data).toEqual({
+          MetricData: [{
+            Dimensions: [{
+              Name: 'environment',
+              Value: 'PROD'
+            }, {
+              Name: 'ExtraDimension',
+              Value: 'Value'
+            }],
+            MetricName: 'metricName',
+            Unit: 'Count',
+            Timestamp: jasmine.any(Number),
```

Comment:
Could also verify that it's a number that's `>=` the value of `Date.now()` when the test case started